misago/users/management/commands/removeips.py

```diff
@@ -0,0 +1,18 @@
+from django.contrib.auth import get_user_model
+from django.core.management import BaseCommand, CommandError
+
+from misago.conf import settings
+from misago.core.utils import ANONYMOUS_IP
+from misago.users.signals import anonymize_old_ips
+
+class Command(BaseCommand):
+    help =  "Anonymizes users IPs stored for longer than set in MISAGO_IP_STORE_TIME."
+    
+    def handle(self, *args, **options):
+      if not settings.MISAGO_IP_STORE_TIME:
+        self.stdout.write("IP anonymization is disabled")
+        return
+      
+      anonymize_old_ips.send(sender=self)
+      self.stdout.write("IP addresses older than {:d} days have been anonymized!".format(not settings.MISAGO_IP_STORE_TIME))
```

Comment:
`format(not settings.MISAGO_IP_STORE_TIME)` is wrong here. It will cause message to display incorrectly. Also `{:d}` is unnecessary. The correct line would be:

```python
self.stdout.write("IP addresses older than {} days have been anonymized!".format(settings.MISAGO_IP_STORE_TIME))
```

Also lets put blank line between self.stdout.wrtie and `anonymize_old_ips` for extra logic separation.